backend/src/test/java/gov/cdc/usds/simplereport/service/ReminderServiceTest.java

```diff
@@ -115,10 +115,10 @@ void backdateQueuedOrgCreatedAt(OrganizationQueueItem queuedOrg) throws SQLExcep
     // verification reminder email will be sent to.
     String query =
         "UPDATE simple_report.organization_queue SET created_at = (NOW() + AGE(NOW() AT TIME ZONE 'America/New_York', NOW()) - INTERVAL '1 DAY')::date + INTERVAL '12 hour' WHERE internal_id = ?";
-    Connection conn = _jdbc.getDataSource().getConnection();
-    PreparedStatement statement = conn.prepareStatement(query);
-    statement.setObject(1, queuedOrg.getInternalId());
-    statement.execute();
-    conn.close();
+    try (Connection conn = _jdbc.getDataSource().getConnection()) {
```

Comment:
This was a potential resource leak: the `try` block here is actual a special [`try-with-resources`](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)

https://github.com/CDCgov/prime-simplereport/security/code-scanning/52